runtime/src/bank.rs

```diff
@@ -88,11 +88,14 @@ pub struct Bank {
     last_id_queue: RwLock<LastIdQueue>,
 
     /// Previous checkpoint of this bank
-    parent: Option<Arc<Bank>>,
+    parent: RwLock<Option<Arc<Bank>>>,
 
-    /// Hash of this Bank's state. Only meaningful after freezing it via `new_from_parent()`.
+    /// Hash of this Bank's state. Only meaningful after freezing it
     hash: RwLock<Hash>,
 
+    /// Hash of this Bank's parent's state
+    parent_hash: Hash,
```

Comment:
Looks like our PRs crossed in flight.  Can you rebase on this: https://github.com/solana-labs/solana/pull/2853